google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/GCRulesTest.java

```diff
@@ -280,6 +280,97 @@ public void intersectionOfUnions() {
     assertThat(actual.toProto()).isEqualTo(expected);
   }
 
+  @Test
+  public void multipleSetters() {
+    com.google.protobuf.Duration seconds =
+        com.google.protobuf.Duration.newBuilder().setSeconds(1).build();
+    // Only the last setXXX() value persists after build().
+    GcRule proto_gcRule = GcRule.newBuilder().setMaxNumVersions(5).setMaxAge(seconds).build();
+
+    GCRules.GCRule model_gcRule = GCRULES.maxAge(Duration.ofSeconds(1));
+    assertThat(GCRULES.fromProto(proto_gcRule)).isEqualTo(model_gcRule);
+  }
```

Comment:
I'm having a hard time understanding what this is testing.  The protobuf behavior for one-of fields?

Reply:
Tests that the multiple setters override existing value... It was to confirm my understanding of protobuf behavior. I think this could be removed.

Reply:
Yea, I don't think testing underlying protobuf behavior should happen in this repo